lib/rules/prefer-find-by.ts

```diff
@@ -111,6 +111,226 @@ export default createTestingLibraryRule<Options, MessageIds>({
       });
     }
 
+    function getWrongQueryNameInAssertion(
+      node: TSESTree.ArrowFunctionExpression
+    ) {
+      if (
+        !isCallExpression(node.body) ||
+        !isMemberExpression(node.body.callee)
+      ) {
+        return null;
+      }
+
+      // expect(getByText).toBeInTheDocument() shape
+      if (
```

Comment:
Wouldn't `isPresenceAssert` and `isAbsenceAssert` be helpful here for some of the current checks?

Reply:
Hm, I am not sure how they'd fit in here 🤔

Reply:
Ah sorry, I saw them used later. It wasn't sure what was the purpose of this function the first time I went through the PR.